src/utility_belt/validation.clj

```diff
@@ -0,0 +1,9 @@
+(ns utility-belt.validation)
+
+
+;; Kudos to helpful people on the internets, including SO and IRC
+(def email-pattern-string
+  "[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?")
```

Comment:
ha blink has one as well `#"^[^\s@]+@[^\s@]+\.[^\s@]+$"` :D

Reply:
Yeah, I think if we switch to using this regex we will see some interestingly invalid data